css/style.css

```diff
@@ -51,64 +73,134 @@ body {
     position: absolute;
     font-weight: 700;
     font-size: 1.5rem;
-    text-transform: uppercase;
 }
 
 .resume_buttons_container {
     margin-top: 250px;
 }
 
-.game_title {
-    display: block;
-}
-
-.start_buttons_container,
-.resume_buttons_container {
-    display: block;
-    width: 300px;
-    position: relative;
-    left: 450px;
-}
-
 .buttons {
     display: block;
+    width: 250px;
+    position: relative;
     text-align: center;
     line-height: 50px;
     border: 3px solid #fff;
+    border-radius: 5px;
+    background-color: rgba(255, 0, 0, .7);
```

Comment:
colour is defined using rgba here and a 3 character hex code 2 lines above. please choose one format for consistency